extensions/wikia/Rail/styles/RailElement.scss

```diff
@@ -0,0 +1,262 @@
+@import "skins/shared/color";
+
+$black: #000;
+$white: #FFF;
+$color-divider: #E6EBF2;
+$color-rail-background: mix($color-page, $black, 90%);
+@if $is-dark-wiki {
+    $color-rail-background: mix($color-page, $white, 90%);
+}
+
+
+@mixin wikia-rail-element() {
+    background-color: $color-rail-background;
+    margin-bottom: 10px;
+    padding: 20px;
+    position: relative;
+    word-wrap: break-word;
+    zoom: 1;
```

Comment:
As far as I remember zoom: 1 was some sort of a hack for IE. Do we still need it?